src/SilverlightWCFRIA/SilverlightWCFRIA.csproj

```diff
@@ -83,6 +93,12 @@
 
   <ItemGroup>
     <Folder Include="Generated_Code\" />
+    <Folder Include="Models\" />
+    <Folder Include="Helpers\" />
   </ItemGroup>
-  
+
+  <ItemGroup>
+    <UpToDateCheckInput Remove="Views\About.xaml" />

```

Comment:
We do not need this. That is a strange behavior of Visual Studio. It constantly creates this config.